src/context/directory/index.ts

```diff
@@ -78,14 +78,17 @@ export default class DirectoryContext {
     // Copy clients to be used by handlers which require converting client_id to the name
     // Must copy as the client_id will be stripped if AUTH0_EXPORT_IDENTIFIERS is false
     //@ts-ignore because assets haven't been typed yet TODO: type assets
-    this.assets.clientsOrig = [...this.assets.clients];
+    this.assets.clientsOrig = [...this.assets.clients || []];
```

Comment:
Clients could theoretically be excluded and thus be empty so need to default to empty array.